File: beginner/__2000__/2176/2176.py
#mensagem s em bits
bits = [int(x) for x in input()]

bits_1 = 0
for bit in bits:
    if bit == 1:
        bits_1 += 1 
        
if bits_1 % 2 == 0:
    bits.append(0)
else:
    bits.append(1)
    
print("".join(str(n) for n in bits))

File: beginner/__2000__/2176/2176.js
const fs = require("fs")
const input = fs.readFileSync("/dev/stdin", "utf8").split("").map(Number)

let bits_1 = 0
for (const bit of input){
    bits_1 += bit == 1 ? 1 : 0
}

input.push(bits_1 % 2 == 0 ? 0 : 1)

console.log(input.join(""))